web/packages/teleterm/package.json

```diff
@@ -30,6 +30,7 @@
     "split2": "4.2.0",
     "strip-ansi": "^7.1.0",
     "tar-fs": "^3.0.6",
+    "which": "^4.0.0",
```

Comment:
I know it's a small package, but maybe we could add `@types/which`?

Reply:
Done, mostly to check out the flow of adding a new package with pnpm. ;)